laddertools/ragl.py

```diff
@@ -218,6 +219,10 @@ def _main(args):
     if playoffs:
         _handle_extra_outcomes(c, extra_outcomes, playoffs)
 
+    if 'Forfeit_Games' in players_info.keys():
+        forfeit_games = [game for game in players_info['Forfeit_Games']]
```

Comment:
Why can't you use `players_info['Forfeit_Games']` directly? If it really needs to be a list you can use `list(players_info['Forfeit_Games'])` but I'm not sure you need that.

Reply:
Yep, that worked.